JythonIDE.rst

```diff
@@ -30,9 +30,10 @@ http://pydev.sourceforge.net/ but you won't need to manually download and
 install it.
 
 To install the plugin, start Eclipse and go to the menu :menuselection:`Help -->
-Install new Software...`, type http://pydev.sourceforge.net/updates/ into the
-"Work with" input box and press enter. After a short moment you will see an
-entry for PyDev in the bigger box below. Just select it, clicking on the
+Install new Software...`, type http://www.pydev.org/update_sites/9.3.0/ into the
```

Comment:
Can we make this `https:` please? It worked for me and seems a good idea, especially as the content is unsigned.

Reply:
I have changed the url to https. And I have also updated the corresponding screenshot. I have tested the url with a fresh installation of Eclipse Version 2023-03 (4.27.0), using OpenJDK 17